mlir/unittests/IR/AffineMapTest.cpp

```diff
@@ -21,3 +21,44 @@ TEST(AffineMapTest, inferMapFromAffineExprs) {
   map.replace(replacements);
   EXPECT_EQ(map, map);
 }
+
+TEST(AffineMapTest, isProjectedPermutation) {
+  MLIRContext ctx;
+  OpBuilder b(&ctx);
+
+  // 1. Empty map - a projected permutation.
```

Comment:
its a 'projected permutation' test and TRUE, FALSE tells the expected so no need to repeat 'a projected permutation'

Reply:
Good suggestion, thanks!